corehq/apps/style/static/style/js/multiselect_utils.js

```diff
@@ -1,6 +1,6 @@
 /* global django:false _:false */
 
-hqDefine('style/js/components/multiselect_utils', function () {
+hqDefine('style/js/multiselect_utils', function () {
```

Comment:
just curious - is there a convention around how to use `/components/` i should know about? just wondering why you moved this one but not others like `inline_edit.js`